setup.cfg

```diff
@@ -44,6 +44,12 @@ norecursedirs =
     measure_extinction/docs/_build/*
     measure_extinction/docs/api/*
 
+[options.entry_points]
+console_scripts =
+    meplot_spec = measure_extinction.plotting.plot_spec:main
+    meplot_ext = measure_extinction.plotting.plot_ext:main
+    mecalc_ext = measure_extinction.utils.calc_ext:main
```

Comment:
Just curious, what exactly is the purpose of this?

Reply:
These mean that if you have pip installed this package, then you can use `meplot_spec` etc. on the command line from any directory.  Just easier to call the command line scripts then having to remember their full paths.

Reply:
Cool! Does that work in a script as well, when importing those modules?

Reply:
Not quite sure what you mean.

Reply:
So, you say you can use those shortcuts on the command line to call the scripts. My question is if you can also do
`import meplot_spec` in another python script, instead of
`import measure_extinction.plotting.plot_spec` e.g.

Reply:
No.   This only works for the command line.  Imports have to use the full package info.